doc/api/magic-plot.md

```diff
@@ -293,3 +293,52 @@ You can also show the percentage on top of the pie using the `S`/`show-numbers`
 ```{code-cell} ipython3
 %sqlplot pie --table penguins.csv --column species -S
 ```
+
+## Parametrizing arguments
+
+JupySQL supports variable expansion of arguments in the form of `{{variable}}`. This allows the user to specify arguments with placeholders that can be replaced by variables dynamically.
+
+Currently, `table`, `column`, `schema`, `with` and `orient` arguments of `%sqlplot` can be parametrized. Let's see an example:
```

Comment:
allows parametrization of string arguments. So list is valid but removed this line completely because everytime a new argument is added we need to remember to add to this list: https://github.com/ploomber/jupysql/pull/941/commits/f26bccfb5676081d60845f0042bd551c5c784d62#diff-d2bac2aa48fbf4cb4aee34358d99fb9653fe443d5d3359ea2848dd36bdaa2f3bL301-L302